civictechprojects/forms.py

```diff
@@ -81,7 +81,7 @@ def create_project(request):
     def delete_project(request, project_id):
         project = Project.objects.get(id=project_id)
 
-        if not request.user.username == project.project_creator.username:
+        if not (request.user.username == project.project_creator.username or request.user.is_staff):
```

Comment:
Now that this logic is becoming more complex, I'd recommend a couple things.

- Put this owner/staff check in a function
- Use parentheses to separate the logical clauses, e.g. ((x == y) or z)

Reply:
Moved logic to a function in form_helpers.py.